scripts/ae2.zs

```diff
@@ -318,11 +318,11 @@ recipes.addShaped("wct", <wct:wct>, [
 ]);
 
 // Creative Wireless Crafting Terminal
-//recipes.removeByRecipeName("wct:wct");
-//recipes.addShaped("wct_creative", <wct:wct_creative>, [
-//    [<wct:wct>, <metaitem:field.generator.luv>],
-//    [<appliedenergistics2:part:16>, <ore:gtceScrewdrivers>]
-//]);
+recipes.removeByRecipeName("wct:wct");
+recipes.addShaped("wct_creative", <wct:wct_creative>, [
+    [<wct:wct>, <metaitem:field.generator.luv>],
```

Comment:
We decided to push the tier of this recipe further. Replace the field generator with a `uv` tier one.

Reply:
At that point would it even be worth crafting at all? To my knowledge, the only difference between a Creative WCT and Infinity-Boosted WCT is the unlimited energy, which is hardly relevant at LuV+.

Reply:
I made the change anyway, for now.